spice/spice.py

```diff
@@ -428,6 +432,9 @@ async def get_response(
                     chat_completion = await client.get_chat_completion_or_stream(call_args)
                     text, input_tokens, output_tokens = client.extract_text_and_tokens(chat_completion)
 
+                if converter:
+                    text = converter(text)
```

Comment:
Thanks! I actually wanted to convert after validation. And if conversion throws an exception  (imagine a Value Error from pydantic) then this should be treated the same as a failed validation.

Can you also change SpiceResponse to hold the generic result which is gotten from the converter in addition to text?